reduce_size.py

```diff
@@ -0,0 +1,117 @@
+import concurrent.futures
+import datetime
+import pandas as pd
+from glob import glob
+from tardis_dev import datasets, get_exchange_details
+import os
+import argparse
+
+COLUMNS = ["symbol", "timestamp", "type", "strike_price", "expiration", "underlying_price", "bid_price", "bid_amount",
+           "ask_price", "ask_amount", "mark_price", "mark_iv", "delta", "theta"]
```

Comment:
i dont recall are other greeks provided? such as vega or gamma?

Reply:
These are the default columns debrit gives 

```
Index(['exchange', 'symbol', 'timestamp', 'local_timestamp', 'type',
       'strike_price', 'expiration', 'open_interest', 'last_price',
       'bid_price', 'bid_amount', 'bid_iv', 'ask_price', 'ask_amount',
       'ask_iv', 'mark_price', 'mark_iv', 'underlying_index',
       'underlying_price', 'delta', 'gamma', 'vega', 'theta', 'rho'],
      dtype='object')
```

Reply:
i think keeping the gamma, vega and theta would be good :)